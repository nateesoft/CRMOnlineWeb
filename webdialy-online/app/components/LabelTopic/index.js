import React from 'react';
import PropTypes from 'prop-types';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(() => ({
  updateItemHeader: {
    border: '1px solid orange',
    boxShadow: '4px 2px',
    padding: '5px',
    width: '100%',
    textAlign: 'center',
  },
}));

export default function LabelTopic(props) {
  const classes = useStyles();

  return (
    <Typography variant="h5" className={classes.updateItemHeader}>
      {props.children}
    </Typography>
  );
}

LabelTopic.propTypes = {
  children: PropTypes.any,
};
