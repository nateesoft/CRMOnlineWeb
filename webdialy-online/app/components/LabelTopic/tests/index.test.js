import React from 'react';
import renderer from 'react-test-renderer';

import LabelTopic from '../index';

describe('LabelTopic components', () => {
  it('test snapshot', () => {
    const component = renderer.create(<LabelTopic />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
