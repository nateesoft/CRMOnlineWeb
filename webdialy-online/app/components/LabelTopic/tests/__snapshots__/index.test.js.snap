// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelTopic components test snapshot 1`] = `
<h5
  className="MuiTypography-root makeStyles-updateItemHeader-1 MuiTypography-h5"
/>
`;
